src/Assignment-2/assignment2.html

```diff
@@ -12,7 +12,111 @@
 	<div id="container">
 
 	<!-- Start your code from here -->
-	
+    <div class="main-body">
+        
+        <!-- START Header -->
+        <div>
+            <header>
+                <div class="first-part">
+                    <div class="item">
+                        <img src="https://www.the10xacademy.com/assets/images/logo-color.png" alt="">
+                    </div>
+                    <div class="item">  
+                        <form action="">
+                            <input type="text" placeholder="Search" name="Search">
+                        </form>
+                    </div>
+                </div>
+                <div class="second-part">
+                    <nav class="nav">
+                        <div class="nav-link"><a href="https://en.wikipedia.org/wiki/Tiger">Wikipedia </a></div>
+                        <div class="nav-link"><a href="https://www.google.com/search?q=hd+images+of+tiger&sxsrf=AOaemvIGdyxSW_BXzv9cFH7W2u8M2wDxbw%3A1636607337874&ei=aaWMYfLXNMSy9QPU3I3YBA&oq=hd+images+of+tiger&gs_lcp=Cgdnd3Mtd2l6EAMyBQgAEIAEMgUIABCABDIGCAAQFhAeMgYIABAWEB4yBggAEBYQHjIGCAAQFhAeMgYIABAWEB4yBggAEBYQHjIGCAAQFhAeMggIABAWEAoQHjoHCCMQ6gIQJzoECCMQJzoLCAAQgAQQsQMQgwE6BQguEIAEOggIABCABBCxAzoECAAQQzoGCAAQChBDOg0ILhCxAxDHARCjAhBDOggIABCxAxCDAToNCAAQgAQQhwIQsQMQFDoECC4QQzoHCAAQsQMQQzoKCAAQgAQQhwIQFEoECEEYAFAAWKkvYN4yaAJwAngBgAHkBYgBpjeSAQoyLTMuMTYuNi0xmAEAoAEBsAEKwAEB&sclient=gws-wiz&ved=0ahUKEwiysIq4xY_0AhVEWX0KHVRuA0sQ4dUDCA4&uact=5">Hd Images</a></div>
+                        <div class="nav-link"><a href="https://en.wikipedia.org/wiki/White_tiger">White Tiger</a></div>
+                        <div class="nav-link"><a href="https://en.wikipedia.org/wiki/Black_Panther_(film)">Black Panther Wiki</a></div>
+
+                    </nav>
+                </div>
+            </header>
+        </div>
+        <!-- END Header -->
+    <hr>
+
+        <!-- START BODY -->
+        <section class="main-section">
+            <div class="left-section">
+                    <aside>
+                        <ul>Section 1</ul>
+                        <ul>Section 2</ul>
+                        <ul>Section 3</ul>
```

Comment:
work on format of usage of ul tag.if its list item we need to use li tag inside ul tag

> 				<ul>
					<li>section 1</li>
					<li>section 2</li>
					<li>section 3</li>
					<li>section 4</li>
					<li>section 5</li>
					<li>section 6</li>
					<li>section 7</li>
					<li>section 8</li>
					<li>section 9</li>
					<li>section 10</li>
				</ul>